index.html

```diff
@@ -69,5 +70,185 @@ <h2 class="text-2">
         
     </section>
 
+    <section class="wrapper-works">
+        <header class="title">
+            My Recent Works
+        </header>
+        
+        
+        <img src="./images/img-work-mobile.png" class="image-work">
+
+        <h3 class="title-multi">
+            Multi-Post Stories
+        </h3>
+
+        <p class="works-description">
+            A daily selection of privately personalized 
+                reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text
+                ever since the 1500s, when an unknown
+                printer took a standard dummy text.
+        </p>
+
+        
+        <ul class="button-container">
+            <li class="button1">css</li>
+            <li class="button1">html</li>
+            <li class="button1">bootstrap</li>
+            <li class="button1">Ruby</li> 
+            <button class="button2">See Project</button>                                  
+        </ul>   
+
+    </section>
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest you include the works cards in this section since they belong to the same logical section of your page! 🤓 🛠️